### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by RITHINTHOTA**

### CODE REVIEW

**Comment by thotarithin** [comment]
why to add oil to karam dosa ? It's already there ...

**Comment by RITHINTHOTA** [comment]
> why to add oil to karam dosa ? It's already there ...

extra oil enhances the taste of karam dosa

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (thotarithin) [inline menu.txt:3]: why oil is added at the end?

Comment (RITHINTHOTA) [reply menu.txt:3]: as it is added as extra

**Review by thotarithin — APPROVED** [review]
Everything is good. You can proceed

**Pull request merged**